app/js/main.js

```diff
@@ -328,17 +333,55 @@ function startup() {
 
   // Preview a captured frame
   frameReelRow.addEventListener("click", function(e) {
-    if (e.target.className === "frame-reel-img") {
-      if (winMode !== "playback") {
-        switchMode("playback");
+    if (e.target.className.includes("frame-reel-img")) {
+      // For an unselected, non-keyframe
+      if (!e.target.className.includes("selected")) {
+        // Remove previous selection
+        _removeFrameReelSelection();
+
+        // Highlight the clicked image
+        e.target.classList.add("selected");
+        if (winMode !== "playback") {
+          switchMode("playback");
+        }
+
+        // Display the selected frame
+        var imageID = parseInt(e.target.id.match(/^img-(\d+)$/)[1], 10);
+        _displayFrame(imageID);
```

Comment:
Go ahead and rewrite these into one line, as this is the only time `imageID` is used.